main/app-menu.js

```diff
@@ -1,26 +1,42 @@
-const { Menu, MenuItem, ipcMain } = require('electron')
+const fs = require('node:fs/promises')
+const { Menu, MenuItem, ipcMain, dialog, BrowserWindow } = require('electron')
 const { ipcMainEvents } = require('./ipc')
+const { getLog } = require('./saturn-node')
 
 function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
-  // Add "Check for updates..." item to the Application menu on MacOS
-  // GitHub issues tracking the work to add this menu item on other platforms:
-  //  - Windows: https://github.com/filecoin-project/filecoin-station/issues/63
-  //  - Linux: https://github.com/filecoin-project/filecoin-station/issues/64
-  if (process.platform !== 'darwin') return
-
   const menu = Menu.getApplicationMenu()
   if (!menu) return
 
-  menu.items[0].submenu?.insert(1, new MenuItem({
-    id: 'checkForUpdates',
-    label: 'Check For Updates...',
-    click: () => { ctx.manualCheckForUpdates() }
-  }))
-  menu.items[0].submenu?.insert(2, new MenuItem({
-    id: 'checkingForUpdates',
-    label: 'Checking For Updates',
-    enabled: false,
-    visible: false
+  if (process.platform === 'darwin') {
+    // Add "Check for updates..." item to the Application menu on MacOS
+    // GitHub issues tracking the work to add this menu item on other platforms:
+    //  - Windows: https://github.com/filecoin-project/filecoin-station/issues/63
+    //  - Linux: https://github.com/filecoin-project/filecoin-station/issues/64
+    menu.items[0].submenu?.insert(1, new MenuItem({
+      id: 'checkForUpdates',
+      label: 'Check For Updates...',
+      click: () => { ctx.manualCheckForUpdates() }
+    }))
+    menu.items[0].submenu?.insert(2, new MenuItem({
+      id: 'checkingForUpdates',
+      label: 'Checking For Updates',
+      enabled: false,
+      visible: false
+    }))
+  }
+
+  menu.items[1].submenu?.insert(1, new MenuItem({
+    label: 'Save Module Logs As…',
```

Comment:
It gets inserted on `menu.items[1]`, which is the `File` menu. This is how it looks on macOS:

<img width="467" alt="Screenshot 2022-09-06 at 13 12 37" src="https://user-images.githubusercontent.com/10247/188622041-03087ae1-c738-4866-9cec-77b200e55c7c.png">
 
I think this will work on other OSs too, I'm still setting up my ubuntu and windows VMs for testing.

Reply:
I've added the comments 👍  And agreed about the position, this was also my thinking